docs/Tutorials/Create-IBFT-Network.md

```diff
@@ -23,154 +23,134 @@ consensus protocol](../Consensus-Protocols/IBFT.md).
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "alloc": {
+       "fe3b557e8fb62b89f4916b721be55ceb828dbd73": {
+          "privateKey": "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+          "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+          "balance": "0xad78ebc5ac6200000"
+       },
+       "627306090abaB3A6e1400e9345bC60c78a8BEf57": {
+         "privateKey": "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+         "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+         "balance": "90000000000000000000000"
+       },
+       "f17f52151EbEF6C7334FAD080c5704D77216b732": {
+         "privateKey": "ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f",
+         "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
+         "balance": "90000000000000000000000"
+       }
+	  }
+ },
+ "blockchain": {
+   "nodes": {
+     "generate": true,
+       "count": 4
+   }
+ }
+}
 ```
 
+### 3. Generate Node Keys and Genesis File 
 
-### 4. RLP Encode Extra Data 
-
-The `extraData` property in IBFT 2.0 genesis files is an RLP encoding of `[32 Bytes Vanity, List<Validators>, No Vote, Round=Int(0), 0 Seals]`. 
-
-In the `IBFT-Network` directory, use the Pantheon subcommand [`rlp encode`](../Reference/Pantheon-CLI-Syntax.md#rlp) to generate the `extraData` 
-RLP string to include in the genesis file. 
+In the `IBFT-Network` directory, generate the node key and genesis file: 
 
 ```bash tab="MacOS"
-pantheon rlp encode --from=toEncode.json --to=rlpEncodedExtraData
-```
-
-```bash tab="Windows"
-pantheon rlp encode --from=toEncode.json --to=rlpEncodedExtraData
+pantheon operator generate-blockchain-config --config-file=ibftConfigFile.json --to=networkFiles
 ```
 
+In the `networkFiles` directory, the following are created: 
+* `genesis.json` - genesis file including the `extraData` property specifying the four nodes are validators
+* Directory for each node named with the node address and containing the public and private key for each node 
 
-### 5. Create Genesis File 
-
-The genesis file defines the genesis block of the blockchain (that is, the start of the blockchain).
-The [IBFT 2.0 genesis file](../Consensus-Protocols/IBFT.md#genesis-file) includes the address of Node-1 as the initial validator in the
-RLP encoded `extraData` string.    
-
-All nodes in a network must use the same genesis file. 
+!!! example 
+    ADD DIRECTORY STRUCTURE HERE
+    
+### 4. Copy Node Private Keys to Node Directories 
 
-Copy the following genesis definition to a file called `ibftGenesis.json` and save it in the `IBFT-Network` directory: 
+For each node, copy the `key.priv` file to the `data` directory for that node and rename `key`. (DOES IT HAVE TO BE RENAMED?)
```

Comment:
The output file name can be specified in CLI args

Reply:
Fixed